crates/devolutions-pedm/src/elevator/virtual_account_elevator.rs

```diff
@@ -51,45 +53,49 @@ impl VirtualAccountElevator {
         }
     }
 
-    fn create_token(&self, token: &Token) -> Result<()> {
-        let virtual_account = create_virtual_account(self.rid, &self.domain, token)?;
+    fn create_token(&self, token: &Token) -> anyhow::Result<()> {
```

Comment:
thought: I’m not sure the name is very good, as "create_token" suggests the output of the function would be a `Token`, but it’s not. Seems to be more about "upgrading" or "elevating" an existing token?

Reply:
The naming is bad; but it is creating a token. Let's leave for now as you suggest, because I don't have a better idea